datastore/gcloud/aio/datastore/constants.py

```diff
@@ -53,6 +53,7 @@ class PropertyFilterOperator(enum.Enum):
     LESS_THAN_OR_EQUAL = 'LESS_THAN_OR_EQUAL'
     NOT_EQUAL = 'NOT_EQUAL'
     UNSPECIFIED = 'OPERATOR_UNSPECIFIED'
+    IN = 'IN'
```

Comment:
Please update the comment at the top of this class (if it's trivial to add `NOT_IN` as well, that'd be appreciated!)

nit: please keep this enum sorted.